experiments.json

```diff
@@ -1,15 +1,46 @@
 {
-  "onboarding-a": {
-    "match": {},
+  "search-term": {
+    "match": {
+      "version": "^4[6-9]\\..*"
```

Comment:
Do we need to include this? Technically this won't do anything for earlier releases since the code isn't in the client.